<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowSignalSetEditor</class>
 <widget class="QDialog" name="WindowSignalSetEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gb_SelectedSet">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>321</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Currently selected set:</string>
   </property>
   <widget class="QLabel" name="lbl_NameTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Signals">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Signals:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Save">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Create new</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lb_Signals">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddSignal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Signal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_RemoveSignal">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>340</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Signal</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_Track">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Selected track:</string>
    </property>
    <widget class="QPushButton" name="btn_TrackSelect">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_TrackName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_AvailableSets">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Available sets:</string>
   </property>
   <widget class="QListWidget" name="lb_AvailableSets">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_DeleteSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete selected Signal Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Select">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>SELECT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
